packages/features/ee/organizations/context/provider.ts

```diff
@@ -22,19 +22,19 @@ export type OrganizationBranding =
 /**
  * Allows you to access the flags from context
  */
-const OrganizationBrandingContext = createContext<OrganizationBranding | null>(null);
+const OrganizationBrandingContext = createContext<{ orgBrand: OrganizationBranding } | null>(null);
 
 /**
  * Accesses the branding for an organization from context.
- *
  * You need to render a <OrgBrandingProvider /> further up to be able to use
  * this component.
+ *
+ * @returns `undefined` when data isn't available yet, `null` when there's an error, and the data(which can be `null`) when it's available
  */
 export function useOrgBranding() {
   const orgBrandingContext = useContext(OrganizationBrandingContext);
-  if (orgBrandingContext === null)
```

Comment:
The value can itself be null if the user isn't an org member. So, make a wrapper object that has orgBrand and that wrapper objet can be checked if it's nullish or not.